resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
How would we handle snapshots then? Or are we only going to push images on every release, at which point we update all of the Dockerfiles?